probes/src/main/resources/private/probe-fogbow.conf

```diff
@@ -1,14 +1,11 @@
-service_success_rate_probe_id=1
-resource_availability_probe_id=2
-service_latency_probe_id=3
-service_reachability_probe_id=4
-
 ### Required prefix 'http://' ###
+probe_target=http://localhost
+target_docker_port=5555
+fta_address=http://localhost:8090/observation
```

Comment:
The defaults aggregator's port is 8088 and the endpoint `/metric`.

Reply:
Done.